pkg/controllers/rollout/controller.go

```diff
@@ -397,20 +398,27 @@ func (r *Reconciler) pickBindingsToRoll(ctx context.Context, allBindings []*flee
 			} else {
 				canBeReadyBindings = append(canBeReadyBindings, binding)
```

Comment:
if the bindingReady then it can't be failed, so shouldn't this be in line 388 block?

Reply:
I think it can be in line 389 in the else block, but the current flow also makes sense. A binding which is not ready but still hasn't failed is a canBeReady binding from what I understand